tests/test_deployments/test_app.py

```diff
@@ -88,6 +107,8 @@ def setup_class(cls) -> None:
     @classmethod
     def teardown_class(cls) -> None:
         """Teardown the test."""
+        os.environ.clear()
+        os.environ.update(cls._os_env)
```

Comment:
yeah... should have put that there when i wrote it the first time. `monkeypatch` fixture offer no solution, even if used as an autouse fixture it will not be available in `setup_class`. May be [a way to do it](https://stackoverflow.com/questions/42332030/pytest-monkeypatch-setattr-inside-of-test-class-method), to be investigated at a later date